### ISSUE TITLE
[ADE Research - Feedback] - Sitewide Facilities: Facility Locator Mobile Map (Missing search radius on list view)

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#107226]
> [!NOTE]
>  This is the child ticket of #105562 - created by ADE to track an accessibility issue found, during a pilot session or user research session.

### Issue Identified
- **Name of Issue:** Missing search radius
- **Description of issue encountered:** Participant wanted to be able to change the search radius on both list and map views. 
   - Currently, can only change search radius on Map view by zooming in or out and clicking “Search this area”. Participant found this unintuitive.
   - Accessibility note - the same functionality should be available on both views, especially important for SR users who are very unlikely to use the map view 
- **Images or Links:** n/a

### Number of Issue Occurrences
| **Research** | **Date of Research** | **Browser** | **Assistive Tech** | **Device Type** | **P#** | **Shared in Feedback Channel** | 
|------------------------|------------------|------------|--------------------|-----------------|-------|-------|
| `User Session`                       |      03/04/2025    | `Chrome`     |        none  |  `Mobile`             |      `P1` | `Yes`         |        
| `User Session` or `Pilot Session`                       |      `xx`/`xx`/2025     |`Safari`, `Chrome`, `Firefox`, `Edge`, etc.          |        `VoiceOver, TalkBack, Dragon, etc.`  |  `Mobile`, `Tablet` or `Desktop`            |       `P#`  | `Yes`,  `No` or `Unknown`        |        
| `User Session` or `Pilot Session`                       |      `xx`/`xx`/2025    | `Safari`, `Chrome`, `Firefox`, `Edge`, etc.             |        `VoiceOver, TalkBack, Dragon, etc.`   |  `Mobile`, `Tablet` or `Desktop`           |       `P#`  | `Yes`,  `No` or `Unknown`        |        
| `User Session` or `Pilot Session`                       |      `xx`/`xx`/2025      | `Safari`, `Chrome`, `Firefox`, `Edge`, etc.          |        `VoiceOver, TalkBack, Dragon, etc.`   |  `Mobile`, `Tablet` or `Desktop`           |      `P#`  | `Yes`,  `No` or `Unknown`        |        
| `User Session` or `Pilot Session`                       |      `xx`/`xx`/2025       | `Safari`, `Chrome`, `Firefox`, `Edge`, etc.         |        `VoiceOver, TalkBack, Dragon, etc.`   |  `Mobile`, `Tablet` or `Desktop`            |       `P#`  | `Yes`,  `No` or `Unknown`        |        

#### Additional Issue Info: 
- **Is this issue a global issue?**
    - [ ] Yes
    - [x] No
    - [ ] Unknown at this time
 - **Is this issue occurring with multiple types of AT and/or on multiple devices, browsers, etc.?**
    - [x] Yes 
    - [ ] No
    - [ ] Unkown
- **Is this issue a high-priority issue?**
    - [x] Yes
    - [ ] No
    - [ ] Unknown at this time
 - **Type of issue (select all that apply):**
    - [x] Accessibility (required)
    - [x] UX
    - [ ] UI
    - [ ] IA
    - [ ] Content
    - [ ] Other ____   
### Feedback or Recommendations
- If available, please provide feedback and/or recommendation(s), relating to the accessibility issue encountered. 

## Next Steps (ADE Internal)
- [x] **Title** this issue [ADE Research - Feedback] - Team Name: Study (Type of accessibility issue encountered)
- [x] **Verify** ADE, ADE Research and ADE Research Feedback labels are present in ticket.
- [x] **Remind** your team members on this study of this new tracking method for issues.
    - [x] _One ticket per issue, with multiple occurrences in the table._ 
- [x] **Update** the issue count for the study prior to closing this ticket [research tracker](https://docs.google.com/spreadsheets/d/1IBt-C2nXK9uDMPvFdQ_SEEx6WNJ8ms3gKVkuEVlns0U/edit?gid=2141787939#gid=2141787939&range=S:S)
    - [ ] **Reach out to:** @sara-amanda or @lakper with any questions
- [x] **Update** notes template, to reflect this issue has been logged as a ticket.
- [x] **Verify** this ticket's parent is the Research Intake for the study.
- [x] **Complete** all of the GHP fields such as (originator/team; open date; closed date; sprint; effort estimate/actual; priority, last updated, etc.)
- [x] **Validate**: Please move this ticket to validate for @lakper, when done.
- [x] **Close** this ticket c/o @lakper
